### ISSUE TITLE
s.minos.io x86_64 binaries compiled 32-bit?

### ISSUE THREAD

**Issue by kupiakos** [minos-org/minos-static#12]
It seems that the binaries in the x86_64 directory (for example, [bash](http://s.minos.io/archive/bifrost/x86_64/bash-4.4-1.tar.xz)), are compiled 32-bit:

```sh
$ ./static-get -x -a x86_64 bash
bash-4.4-1.tar.xz
bash-4.4-1/
$ file bash-4.4-1/bin/bash
bash-4.4-1/bin/bash: ELF 32-bit LSB  executable, Intel 80386, version 1 (SYSV), statically linked, stripped
```

**Comment by javier-lopez** [comment]
Indeed that's the case, eventually Ive plans to add 64 bit support, however there is a fair amount of work to do it and right now I'm quite busy, I'll keep this report open to notify when it happens

**Comment by kupiakos** [comment]
This should be clear. I would not have expected an explicit x86_64 architecture label to be compiled 32-bit, and had caused me problems. x86_64 should be reported as unsupported, rather than linking directly to i686.

**Comment by Neboer** [comment]
I agree. And if I can do something to help speed up the x86_64 support, I'll be happy to contribute. This is a great project, thanks all the people working on it.

**Comment by javier-lopez** [comment]
@Neboer thank you for your support, this project has been mainly a single person project for some years now, I don't think x86_64 support is arriaving without anyone else taking the lead, I suspect x64 support would involve a new bootstrap base and editing the compile recipes, most of them only support x86. I'm happy to help in the review process, thought.
